Automattic/woocommerce-payments
# Enable documents feature based on is_documents_enabled

Fixes #4017 

#### Changes proposed in this Pull Request

This commit modifies the `is_documents_section_enabled` to take the `is_documents_enabled` account info flag coming from the server to enable the documents feature.

#### Testing instructions

To be tested in conjunction with 1949-gh-Automattic/woocommerce-payments-server.

* Using a US account, refresh the account cache and the page
* Make sure the Documents menu item is hidden even after the refresh
* Switch to an account from the EU, refresh the account cache and the page
* Make sure the Documents menu item is displayed and can be accessed properly after the refresh

-------------------


**Post merge**
